### DISCUSSION TITLE
Subscription management: Is it possible to give a feed the attribute of 'podcast'?

### DISCUSSION THREAD

**Discussion by ghost** [FreshRSS/FreshRSS#5512] [Q&A] [answered]
I would prefer if podcasts were handled differently to normal feeds. Are there any facilities for this?

One thing, for example, would be the option to play the podcast without opening the actual article. Another nice thing would be that if I do open the article, to have the audio play button at the top of the article instead of at the bottom of all the text.

As for letting FreshRSS know that the subscription is a podcast in the first place, it could be a checkbox in the subscription settings for the feed, or automatically inferred from the content of the RSS feed itself.

Edit: if this isn't already possible, maybe it's best addressed by an extension. But it would such a cool headline feature too, in my opinion.

**Comment by Alkarex** [comment, marked as answer]
The concept we have at the moment is to know whether an article has some (multimedia) attachments or not. There is no other field for podcast. But using what we have at the moment, it should be possible for an extension to tweak the UI depending on the presence of attachments and their types
